README.md

```diff
@@ -17,7 +17,7 @@ the frozen files.
 **To build the image:**
 1. Navigate to your project directory.
 1. Copy `DOIRootCA2.cer` into this directory.
-1. Run `docker build --build-arg config=examples/iowa.py --build-arg ref=examples/reference.json -t flood:latest -f Dockerfile-DOI .`. This will create the docker image and name it `flood`.
+1. Run `docker build --build-arg config=examples/iowa.py --build-arg ref=examples/reference.json --build-arg thumbnail=true  -t flood:latest -f Dockerfile-DOI .`. This will create the docker image and name it `flood`.
```

Comment:
Seems wrong to set thumbnail twice essentially, one for generating the data and one for generating the actual thumbnail. I think we could do something similar to what I did to set the FREEZER_BASE_URL (see config.py in project home) using an environment variable. So pass in thumbnail as a build arg, set an environment variable in the docker container to control he value of THUMBNAIL in config.py before running the freeze command. What do you think?